app/controllers/api/stats_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Api::StatsController < Api::BaseController
+  def signups_per_day
+    sql =
+    """
+      select created_at::date, count(*)
+      from publishers
+      where role = 'publisher'
+      group by created_at::date
+      order by created_at::date
+    """
+    result = ActiveRecord::Base.connection.execute(sql).values
+    render(json: fill_in_blank_dates(result).to_json, status: 200)
+  end
+
+  def verified_signups_per_day
```

Comment:
It might be better call this email-verified publishers, since there is no concept of a 'verified publisher' since the multichannel update.  We could also map this to the publisher status.  A publisher is 'onboarding' once they have verified their email address. https://github.com/brave-intl/publishers/blob/e47b83a994d764643f4acecb0190d7bbbc844ddd/app/models/publisher_status_update.rb#L7